chia/_tests/wallet/wallet_block_tools.py

```diff
@@ -37,7 +37,7 @@
     uint8(20),
     bytes(32 * 5),
 )
-DEFAULT_VDF_INFO = VDFInfo(bytes32([0] * 32), uint64(1), ClassgroupElement(bytes(100)))
+DEFAULT_VDF_INFO = VDFInfo(bytes32([0] * 32), uint64(1), ClassgroupElement(bytes100(bytes(100))))
```

Comment:
?

```suggestion
DEFAULT_VDF_INFO = VDFInfo(bytes32([0] * 32), uint64(1), ClassgroupElement(bytes100(100)))
```

Reply:
This is necessary, since `bytes100` does not accept `int` as an argument, whereas `bytes` does. I tried your change and it fails CI because of this.

Reply:
Mm, I did leave that out of the hinting.  Would have been just `SupportsIndex` perhaps, without the `Iterable[]` bit.

https://github.com/Chia-Network/chia-blockchain/blob/1821d6b575d3b588d56495d8a4a0ed445d7b4aed/chia/util/byte_types.py#L30

If we want to continue not allowing that behavior then maybe the `[0] * 100` pattern would be better.  Or `0 for _ in range(100)`.  Or really, `bytes100.zeroes()` and `bytes100.pad()`.  But I didn't get around to adding that before you got around to deleting the whole class. `:]`

Reply:
I didn't delete the class, it was just moved to `chia_rs`. I think this is low priority, but feel free to add it to the type hint there if you want.

Reply:
I think `[0] * 100` is a bit clearer.